packages/renderer/MdToHtml/rules/image.ts

```diff
@@ -11,9 +11,9 @@ function plugin(markdownIt: any, ruleOptions: RuleOptions) {
 		const token = tokens[idx];
 		const src = utils.getAttr(token.attrs, 'src');
 		const title = utils.getAttr(token.attrs, 'title');
+		const alt = token.content;
```

Comment:
No need for the intermediate variable

Reply:
ok I will remove it and directly pass the value